### ISSUE TITLE
very slow read speeds

### ISSUE THREAD

**Issue by wirytiox** [YanWenKun/ComfyUI-Docker#43]
i have an ssd nvme and i get less that 50MB/s reads when loading a model. i am running wsl 2 windows 11. and i have a rtx 3050 8 gb

**Comment by wirytiox** [comment]
it was my bad. i tested read speeds with https://github.com/e7d/docker-diskmark and it was really low. sorry.

**Comment by YanWenKun** [comment]
Yes, bind-mount a directory from NTFS filesystem and "translate" to ext4 would be very slow.

For better I/O, I recommend using a Docker volume or a WSL2 directory for file storage. Although this is less convenient.

If you use root user in the container, this would also avoid "dubious file ownership".

**Comment by wirytiox** [comment]
FOR ANYONE RUNNING INTO THE SAME ISSUE: you have many path. you can convert a drive to ext4 and mount it to wsl (not what i did) or you can transfer all you wsl distros to the desired drive (my choice)
so you run `wsl --shutdown` and move the distro from the windows add or remove programs. then you follow this tutorial for the docker distro https://needlify.com/post/how-to-move-wsl-distributions-including-docker-images-to-new-locations-on-windows-6412384cbd14c
and you should be done. just make a docker volume and it should be stored in the desired drive
